python/feathub/common/utils.py

```diff
@@ -187,3 +188,22 @@ def get_table_schema(table: TableDescriptor) -> Schema:
     for f in table.get_output_features():
         schema_builder.column(f.name, f.dtype)
     return schema_builder.build()
+
+
+def is_local_file_or_dir(url: str) -> bool:
+    """
+    Check whether a url represents a local file or directory.
+    """
+    url_parsed = urlparse(url)
+    return url_parsed.scheme in ("file", "")
+
+
+def get_file_md5(filename: str) -> str:
```

Comment:
Would it be simpler (for the caller code) to replace this method with `get_hive_catalog_identifier`?